moda-contracts/src/ModaRegistry.sol

```diff
@@ -95,10 +113,14 @@ contract ModaRegistry is IModaRegistry, AccessControlEnumerable {
      */
     function addManagers(address[] calldata managers) external {
         for (uint256 i = 0; i < managers.length; i++) {
-            if (_managers[msg.sender].contains(managers[i])) revert DuplicateAddress();
-            if (managers[i] == address(0)) revert AddressCannotBeZero();
-            _managers[msg.sender].add(managers[i]);
-            emit ManagerAdded(msg.sender, managers[i]);
+            if (_managers[msg.sender].contains(managers[i])) {
+                revert DuplicateAddress();
+            }
+            if (managers[i] == address(0)) {
+                revert AddressCannotBeZero();
+            }
+            bool added = _managers[msg.sender].add(managers[i]);
```

Comment:
I believe this ensures uniqueness of addresses, but could be wrong

Reply:
Yep, that's right, fixed